Gemfile

```
@@ -1,8 +1,9 @@
-source 'http://rubygems.org'
-
 source 'https://rubygems.org'
 
-spree_version = 'master'
-gem 'spree', github: 'spree/spree', branch: spree_version
+spree_version = '~> 4.0'
+gem 'spree', spree_version
```

```suggestion
gem 'spree', github: 'spree/spree', branch: 'master'
```

let's not limiting version in Gemfile

---

Agree!